### ISSUE TITLE
I found the savedmodel outputs the logits, whose range is beyond (0,1).Does it is the larger the bettter? Can i use sigmoid func to transform them?

### ISSUE THREAD

**Issue by hazardwayne** [tensorflow/ranking#169]

**Comment by ramakumar1729** [comment]
Yes, larger is better. The value depends on the scoring function you use. You can add a sigmoid transformation on the output of scoring function to get a score in a normalized range.
